.github/workflows/signtx-test.yml

```diff
@@ -63,6 +63,7 @@ jobs:
     - name: Run SignTx Test (ASAN+UBSAN)
       run: |
         cd ${{ github.workspace }}
+        sleep 10
```

Comment:
how do we know that 10 seconds gives us a good safety margin and won't result in somewhat flaky actions? (or do we know that?)

Reply:
guess and check

Reply:
feels like a goldilocks amount. long enough to allow subzero to boot but not too long to delay CI. We can always change it as needed.